### PR TITLE
yacht: implement exercise

### PR DESCRIPTION
**Pull request by mRcfps**
Implement exercise **yacht**. [Canonical Data](https://github.com/exercism/problem-specifications/blob/master/exercises/yacht/canonical-data.json). Discussions are welcome!

### CODE REVIEW

**Comment by cmccandless** [comment]
I would rather not have a new exercise held up by difficulty level discussion... Many exercises' difficulties require reevaluation; we may as well merge this one and add it to the list.

**Comment by mRcfps** [comment]
@cmccandless Of course we'll realize this new exercise first. And when it comes to difficulty reevaluation, I think it's a noteworthy issue for **all** language tracks.

File: exercises/yacht/example.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+def big_straight(dice):
+    return 30 if set(dice) == {2, 3, 4, 5, 6} else 0
+
+
+def yacht(dice):
+    return 50 if len(set(dice)) == 1 else 0
+
+
+categories = {
```
[the remark this answers is not in the document]
Comment (cmccandless) [reply exercises/yacht/example.py]: Enumeration also allows the user to use anything they wish for the constants in their implementation (strings, integers, etc) while still constraining enough for tests to be written.

**Comment by cmccandless** [comment]
@mRcfps merged; thanks for working on this!

**Pull request merged**